pkg/encryption/encryption.go

```diff
@@ -33,78 +33,86 @@ func Increment(nonce *storj.Nonce, amount int64) (truncated bool, err error) {
 }
 
 // Encrypt encrypts data with the given cipher, key and nonce
-func Encrypt(data []byte, cipher storj.Cipher, key *storj.Key, nonce *storj.Nonce) (cipherData []byte, err error) {
+func Encrypt(data []byte, cipher storj.CipherSuite, key *storj.Key, nonce *storj.Nonce) (cipherData []byte, err error) {
 	// Don't encrypt empty slice
 	if len(data) == 0 {
 		return []byte{}, nil
 	}
 
 	switch cipher {
-	case storj.Unencrypted:
+	case storj.EncUnspecified:
 		return data, nil
-	case storj.AESGCM:
+	case storj.EncNull:
+		return data, nil
+	case storj.EncAESGCM:
 		return EncryptAESGCM(data, key, ToAESGCMNonce(nonce))
-	case storj.SecretBox:
+	case storj.EncSecretBox:
 		return EncryptSecretBox(data, key, nonce)
 	default:
 		return nil, ErrInvalidConfig.New("encryption type %d is not supported", cipher)
 	}
 }
 
 // Decrypt decrypts cipherData with the given cipher, key and nonce
-func Decrypt(cipherData []byte, cipher storj.Cipher, key *storj.Key, nonce *storj.Nonce) (data []byte, err error) {
+func Decrypt(cipherData []byte, cipher storj.CipherSuite, key *storj.Key, nonce *storj.Nonce) (data []byte, err error) {
 	// Don't decrypt empty slice
 	if len(cipherData) == 0 {
 		return []byte{}, nil
 	}
 
 	switch cipher {
-	case storj.Unencrypted:
+	case storj.EncUnspecified:
```

Comment:
This is not the correct behavior, it should return an error here.

Reply:
updated